<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NextSampleEditDialog</class>
 <widget class="QDialog" name="NextSampleEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NextSampleEditDialog</string>
  </property>
  <widget class="QLabel" name="lbSampleId">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Sample ID</string>
   </property>
  </widget>
  <widget class="SampleIdLineEdit" name="leSampleId">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>200</width>
     <height>35</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbMode">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>20</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Mode</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbPatientId">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Patient ID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePatientId">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>200</width>
     <height>35</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbName">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>70</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leName">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>70</y>
     <width>200</width>
     <height>35</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbGender">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Gender</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbMale">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>90</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>Male</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbFemale">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>120</y>
     <width>90</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>Female</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbAge">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>120</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Age</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leAgeVal">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>120</y>
     <width>120</width>
     <height>35</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cbAgeUnit">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>120</y>
     <width>60</width>
     <height>35</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbDepartment">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Department</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbDepartment">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>200</width>
     <height>35</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbSender">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>170</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Sender</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbSender">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>170</y>
     <width>200</width>
     <height>35</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbComment">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Comment</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="teComment">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>561</width>
     <height>80</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbOk">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>320</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCancel">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>320</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbPrediluent">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>20</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Prediluent</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbWholeBlood">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>20</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Whole Blood</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbPeripheral">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>20</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Peripheral</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>lineedit.h</header>
  </customwidget>
  <customwidget>
   <class>SampleIdLineEdit</class>
   <extends>LineEdit</extends>
   <header>sampleidlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
